src/firefox/data/scripts/firefox_browser_api.ts

```diff
@@ -52,6 +52,9 @@ class FirefoxBrowserApi implements BrowserAPI {
   }
 
   public showNotification = (notificationText :string) => {
-    // TODO implement!
+    port.emit('showNotification', {
+      text: notificationText,
+      iconURL: './icons/128_online.png'
```

Comment:
The DEFAULT_ICON (??_online.gif) was showing up incredibly terribly in the notification, using this one looked a lot better